cmd/serve/serve.go

```diff
@@ -33,6 +33,7 @@ var (
 	port        string
 	metricsPort string
 	backend     string
+	enableRest  bool
```

Comment:
please change this to enableHttp (or similar)

Reply:
updated